_data/simple-icons.json

```diff
@@ -8226,6 +8226,11 @@
             "hex": "2FCEA0",
             "source": "https://www.skyliner.io/help"
         },
+        {
+            "title": "Skynet",
+            "hex": "00C65C",
```

Comment:
it should be `00C65E` according to https://support.siasky.net/key-concepts/skynet-brand-guidelines

> Skynet's primary color is #00C65E and uses two secondary colors.